### ISSUE TITLE
Can not select text in markdown cell

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#120370]
Issue Type: <b>Bug</b>

Can't select text from a markdown preview, it always tries to drag the cell.

VS Code version: Code 1.55.0 (428daa3650c9b38c92d194172cba735b95a7c77f, 2021-03-26T22:22:39.145Z)
OS version: Darwin x64 20.3.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|4, 4, 4|
|Memory (System)|32.00GB (0.47GB free)|
|Process Argv|--enable-proposed-api donjayamanne.kusto --crash-reporter-id 299c833a-1961-40e9-a094-f10b65ea4e64|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30276681
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
pythonvsnew554:30281908
pythontb:30283811
openwsldoc:30282072
vspre833:30267464
pythonptprofiler:30281270
vshan820:30276952
pythondataviewer:30283453

```

</details>

**Comment by rebornix** [comment]
We can check if this issue exist with custom markdown renderer.

**Comment by roblourens** [comment]
Hm, I don't think we can have both though right?

**Comment by rebornix** [comment]
@roblourens right, it's hard to have both. I'm just wondering if there is a way out. Currently in my personal `setup.ipynb`, I put some scripts in markdown cells which I want to run in terminal sometimes, but right now I can't copy this all. I don't mind turning off drag and drop if possible.

**Comment by roblourens** [comment]
You can put the cell in edit mode

**Comment by schahmatist** [comment]
Exactly the same issue. I cannot select, only can drag, but I do not need to drag. I need to be able to copy text from markdown without entering each time the edit mode. This should be the default behavior and it's as basic as it can get. What's the solution?

**Comment by chengfeng-liu** [comment]
It looks like this issue has not been solved yet. I turned off the drag feature and I now can select the text in rendered markdown cell, but when I copy the selected text, it copied all text in this markdown cell.

How can I solve this? Or any updated versions that handle it?

**Comment by starball5** [comment]
Related on Stack Overflow: [Is there a way to select rendered markdown text in a Jupyter Notebook markdown cell in VS Code?](https://stackoverflow.com/q/77371136/11107541)

I'm not concerned about enabling dragging the cell. There'd still be the left margin to use as a dragging handle, no? That's how I drag non-markdown cells. Even if there wasn't the left margin, couldn't you make it so that holding a keyboard modifier key toggles what happens with the mouse? Ex. the Chrome browser enables selecting link text instead of dragging a link by holding down `alt`.

**Comment by Loccret** [comment]
> It looks like this issue has not been solved yet. I turned off the drag feature and I now can select the text in rendered markdown cell, but when I copy the selected text, it copied all text in this markdown cell.
> 
> How can I solve this? Or any updated versions that handle it?

Without releasing the left mouse button, you can use the copy shortcut (`Ctrl+C`)  to copy specific text from Markdown in VSCode.

**Comment by Norlandz** [comment]
You need to press `ctrl` while selecting the text. Otherwise it deselects the text once mouse releases.
